src/line-graph.js

```diff
@@ -382,12 +399,21 @@ export class LineGraph extends HTMLElement {
           }
 
           if(i >= 6 && i < 12) {
-            l = l + 2;
-            return (i -l) * 20 + 39;
+            return (i - l) * 20 + 39;
```

Comment:
if cases are increasing, so we  can make it "ifelse" ladder or somehow switch

Reply:
for now I will keep this logic and in future switch would be fine!